modules/hello-world/pages/sample-application.adoc

```diff
@@ -1,10 +1,51 @@
 = Sample Application
+:description: Discover how to program interactions with the Couchbase Server via the data, query, and search services -- using the Travel Sample Application with the built-in Travel Sample data Bucket.
 :nav-title: Travel Sample App
 :content-type: tutorial
 :page-topic-type: tutorial
 :page-aliases: ROOT:sample-application,ROOT:tutorial4,ROOT:sample-app-backend
 
-[abstract]
-The Travel Sample Application is currently available in 6 of our SDKs.
+:travel-sample-git-project: try-cb-lcb
+:travel-sample-entrypoint: src/try-cb-lcb.c
 
-The Travel Sample application is available for the xref:3.0@dotnet-sdk:hello-world:sample-application.adoc[.NET]; xref:2.1@go-sdk:hello-world:sample-application.adoc[Go]; xref:3.0@java-sdk:hello-world:sample-application.adoc[Java]; xref:3.0@nodejs-sdk:hello-world:sample-application.adoc[Node.js]; xref:3.0@php-sdk:hello-world:sample-application.adoc[PHP]; and xref:3.0@python-sdk:hello-world:sample-application.adoc[Python] SDKs.
+include::{version-server}@sdk:shared:partial$sample-application.adoc[tag=abstract]
+
+include::{version-server}@sdk:shared:partial$sample-application.adoc[tag=quick-start]
+
+include::{version-server}@sdk:shared:partial$sample-application.adoc[tag=bring-your-own]
+
+include::{version-server}@sdk:shared:partial$sample-application.adoc[tag=using]
+
+
+== Sample App Backend
+
+The backend code shows `libcouchbase` in action with Query and Search,
+but also how to plug together all of the elements and build an application with Couchbase Server and the C SDK.
+
+////
+Here's the airport search code, which checks to see whether the search term for the query string is a three or four letter FAA or ICAO abbreviation,
```

Comment:
How come we left this commented out?

Reply:
C is kinda wordier than Python, I wasn't sure about it :-)

Perhaps @raycardillo can suggest a short snippet that he feels is nicely demonstrative (of *something*, doesn't have to be the same functionality) in libcouchbase? Otherwise just wanted to have *a page* up for now.